clients/da-vinci-client/src/main/java/com/linkedin/davinci/blob/BlobResponse.java

```diff
@@ -0,0 +1,51 @@
+package com.linkedin.davinci.blob;
+
+import java.util.List;
+import java.util.Objects;
+
+
+public class BlobResponse {
+  private boolean isError;
+
+  private String message;
```

Comment:
what does `message` contain?

Reply:
I see that message contains errorMessages. In that case, can we rename this to `errorMessage`?